src/components/Article.jsx

```diff
@@ -0,0 +1,77 @@
+import {useEffect, useState} from 'react';
+import {getArticle, patchArticle, getComments} from '../utils/api';
+import {useParams} from 'react-router';
+import style from '../styles/Article.module.css';
+
+export default function Article() {
+  const [article, setArticle] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+  const [voteCount, setVoteCount] = useState();
+  const [voted, setVoted] = useState(false);
+  const [comments, setComments] = useState();
+  let {article_id} = useParams();
+
+  function dateToString(timestamp) {
+    return new Date(timestamp).toDateString();
+  }
+
+  useEffect(() => {
+    Promise.all([getArticle(article_id), getComments(article_id)]).then(
+      (data) => {
+        console.log('data[0]:', data[1]);
+        setArticle(data[0]);
+        setVoteCount(data[0].votes);
+        setComments(data[1]);
+        setIsLoading(false);
+      }
+    );
+  }, [article_id]);
+
+  const handleVoteSubmit = (e) => {
+    e.preventDefault();
+    setVoted(true);
+    setVoteCount((current) => {
+      return (current += 1);
+    });
+    patchArticle(article_id, {inc_votes: 1});
+  };
+
+  if (isLoading) return;
+
+  return (
+    <>
+      <div className={style.Article__container}>
```

Comment:
It would your code more readable if you had separate ArticleCards and CommentCards. This isn't MVP but something to think about when refactoring. HTML becomes harder to break down when there is a lot of it.

Reply:
I was thinking about that, my intention was to refactor them to separate components when I had the functionality working